tritonus-remaining/src/main/java/org/tritonus/sampled/file/AiffAudioFileReader.java

```diff
@@ -125,7 +130,7 @@ solution is to set the frame size to unspecified (-1).
                 nNumChannels,
                 nFrameSize,
                 fSampleRate,
-                nSampleSize > 8);
+                nSampleSize > 8 && !isLittleEndian);
```

Comment:
it seems adding endianness so easily.
please add test cases that prove specs so far is working w/ new code.